packages/dappmanager/src/modules/installer/ensureIsNotValidatorLegacyPackage.ts

```diff
@@ -0,0 +1,39 @@
+import { InstallPackageData } from "../../types";
+import semver from "semver";
+
+/**
+ * From dappmanager v0.2.46 prevent from installing legacy version (previous to remote signer support) for packages:
+ * - Prysm
+ * - Prysm-Prater
+ * - Gnosis-Beacon-Chain-Prysm
+ */
+export async function ensureIsNotValidatorLegacyPackage(
+  packagesData: InstallPackageData[]
+): Promise<void> {
+  for (const pkg of packagesData) {
+    switch (pkg.dnpName) {
+      case "prysm-prater.dnp.dappnode.eth":
```

Comment:
Move this into a datastructure in src/params.ts

```ts
minimumAllowedPackageVersions: {
  "prysm-prater.dnp.dappnode.eth": "1.0.7"
}
```

Reply:
resolved